simtools/model/array_model.py

```diff
@@ -108,7 +108,7 @@ def _initialize(self, site, array_elements_config):
             array_elements = self._get_array_elements_from_list(
                 site_model.get_array_elements_for_layout(self.layout_name)
             )
-        if array_elements == {}:
+        if not array_elements:
```

Comment:
Suggest to check as well for the empty dict - we need to have array elements to continue.

Reply:
`not array_elements` evaluates to True for any empty dict, list, str etc. This is more general than just a check for empty dict.

Reply:
Yes - you are right!